### ISSUE TITLE
Feature Request: Possibility to communicate offline

### ISSUE THREAD

**Issue by natur960** [oxen-io/session-android#746]
Give users possibility to communicate offline through LAN(wifi)/Bluetooth/BLE
:)

**Comment by User1l0** [comment]
> Give users possibility to communicate offline through LAN(wifi)/Bluetooth/BLE :)

Good idea, i also recommend developers of session to use the briarproject's messenger source code to learn ibriar's bluetooth and local wifi features

**Comment by NiKola-UE** [comment]
Is it possible that no one has commented anything for so long?! 


Yes, support for Bluetooth, Wi-Fi and others would be really welcome, which the authors of Berty already boast about, but their application still lacks a lot. 

As for the source code, I think it is much better and more practical to use 
[CWTCH's](https://git.openprivacy.ca/cwtch.im/) 
instead of Signal's, with Tor's nodes being replaced by Lokinet's. In the case of trying to decrypt, 
[Nymtech's Nym ](https://github.com/nymtech/nym) 
can serve well even though it is still in the early stages of development.
